data/human/news.txt

```diff
@@ -73,9 +73,8 @@ news "merchant anywhere"
 			"The darkest hour is just before the departure."
 			"Don't look a gift shuttle in the engine outlets."
 			"Getting there is half the fun. Getting paid is the other half."
-			"Being a pilot is not all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
-			"We are always a risk of being killed by faulty equipment, solar storms, asteroids, pirates or by angry customers. Ever wonder why you didn't just take a desk job?"
-			"Did you ever look at ships going into hyperspace? They appear to be moving backwards because they are moving faster than light."
+			"Being a pilot isn't all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
+			"We are always at risk of being killed by faulty equipment, solar storms, asteroids, pirates or angry customers. Ever wonder why you didn't just take a desk job?"
```

Comment:
Optional, but this might be smoother:

```suggestion
			"We're always at risk of being killed by faulty equipment, solar storms, asteroids, pirates or angry customers. Ever wonder why you didn't just take a desk job?"
```